src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+class IsoscelesTrapezoid extends Figure {
```

Comment:
The class IsoscelesTrapezoid should implement an interface that defines the draw() method to ensure that the class adheres to a contract that guarantees the presence of that method. For example, if there's an interface named Drawable with a draw() method, this class should implement it.